pkgs/tools/nix/nixos-option/src/libnix-copy-paste.cc

```diff
@@ -2,11 +2,13 @@
 // Since they are not, copy/paste them here.
 // TODO: Delete these and use the ones in the library as they become available.
 
+#include <string_view>
 #include "libnix-copy-paste.hh"
-#include <nix/print.hh>                           // for Strings
+#include <nix/print.hh>
 
+// NOLINTBEGIN
 // From nix/src/nix/repl.cc
-bool isVarName(const std::string_view & s)
+auto isVarName(const std::string_view & s) -> bool
```

Comment:
```suggestion
auto isVarName(std::string_view s) -> bool
```

span-like objects are meant to be used by-value. Otherwise it's a double indirection.